mmv1/products/privateca/api.yaml

```diff
@@ -108,107 +117,248 @@ objects:
+              - !ruby/object:Api::Type::Array
+                name: 'additionalExtensions'
+                description: |
+                  Specifies an X.509 extension, which may be used in different parts of X.509 objects like certificates, CSRs, and CRLs.
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::Boolean
+                      name: 'critical'
+                      required: true
+                      description: |
+                        Indicates whether or not this extension is critical (i.e., if the client does not know how to
+                        handle this extension, the client should consider this to be an error).
+                    - !ruby/object:Api::Type::String
+                      name: 'value'
+                      required: true
+                      description: |
+                        The value of this X.509 extension. A base64-encoded string.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'objectId'
+                      description: |
+                        Describes values that are relevant in a CA certificate.
+                      properties:
+                        - !ruby/object:Api::Type::Array
+                          name: 'objectIdPath'
+                          required: true
+                          item_type: Api::Type::Integer
+                          description: |
+                            An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+              - !ruby/object:Api::Type::Array
+                name: 'policyIds'
+                description: |
+                  Describes the X.509 certificate policy object identifiers, per https://tools.ietf.org/html/rfc5280#section-4.2.1.4.
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::Array
+                      name: 'objectIdPath'
+                      required: true
+                      item_type: Api::Type::Integer
+                      description: |
+                        An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+              - !ruby/object:Api::Type::Array
+                name: 'aiaOcspServers'
+                item_type: Api::Type::String
+                description: |
+                  Describes Online Certificate Status Protocol (OCSP) endpoint addresses that appear in the
+                  "Authority Information Access" extension in the certificate.
+              - !ruby/object:Api::Type::NestedObject
+                name: 'caOptions'
+                description: |
+                  Describes values that are relevant in a CA certificate.
+                properties:
+                  - !ruby/object:Api::Type::Boolean
```

Comment:
We can't `AtLeastOneOf` these, right? Both expect to be able to be omitted?

Reply:
We could add `Required` on is_ca in all cases I think. It would only be settable to `false` in most cases, except when creating a CertificateAuthority, but that is due to API validation

Reply:
And setting false explicitly is the same to the API as not sending the value at all